### PR TITLE
Currency converter in cpp

### PR DESCRIPTION
**Pull request by CodeCinnamon001**
## Assigned Issue Number #16  { if does not exist then create one }
#007

## Language Opted ?
C/C++

## Number of files added or updated ?
3

### Checklist

- [ ✓] I've read the contribution guidelines.
- [ ✓ ] I've referred the correct issue number.
- [ ✓] I've fill up this entire template correctly.

### CODE REVIEW

File: Automations/currency-converter/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+   - Mexican Peso (MXN)
+   - New Zealand Dollar (NZD)
+   - Swedish Krona (SEK)
+
+## Contributing
+
+Contributions are welcome! If you encounter any issues or have suggestions for improvements, please open an issue or submit a pull request.
+
+## Lisence 
+All the rights of this currency converter belongs to (https://github.com/CodeCinnamon001) [CodeCinnamon001]
```
Comment (Arsenic-ATG) [inline Automations/currency-converter/README.md:113]: I don't know if this is intentional or not, but if you like it the current way then it's totally up to you 

```suggestion
All the rights of this currency converter belongs to [CodeCinnamon001](https://github.com/CodeCinnamon001) 
```

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
Thanks for contributing, the program itself looks good to me
There are some minor typos in readme I see that you might wanna look at and after that we can finally merge it.

**Comment by CodeCinnamon001** [comment]
> Thanks for contributing, the program itself looks good to me There are some minor typos in readme I see that you might wanna look at and after that we can finally merge it.

**Comment by Arsenic-ATG** [comment]
Thank you for choosing this project to contribute to 
merging now

**Comment by Arsenic-ATG** [comment]
> Actually English is my second language so i dont know it pretty well. i have committed the changes that you have suggested in readme.md Thanks -CodeCinnamon001

not a problem
English is not my first language either, I use tools like codespell ( https://github.com/codespell-project/codespell ) to detect common typos in documentations and even in code files 🙂

**Pull request merged**